app/uk/gov/hmrc/hmrcemailrenderer/templates/digitalcontact/verifyEmailAddress_cy.scala.txt

```diff
@@ -0,0 +1,13 @@
+@(params: Map[String, Any])Dilyswch eich cyfeiriad e-bost
+
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.txt.salutation(params)
```

Comment:
you should probably use
@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.salutation(params, isWelsh=true)